src/controllers/verifyOTP.js

```diff
@@ -1,15 +1,21 @@
 import { Op } from "sequelize";
-import {User,OTP} from "../database/models";
+import {User,OTP,Blacklist} from "../database/models";
 import verfyToken from "../utils/verifytoken";
 import "dotenv/config"
+import { generateToken } from "../utils/generateToken";
+import { logout } from "../services/user.service";
 const verifyOTP=async(req,res)=>{
     try{
     const token=req.params.token
+    const blacklistToken=await Blacklist.findOne({where:{token}})
```

Comment:
Mhn for best user experience we do not interact with models in controllers and it would be better if controller function is short as possible

Reply:
resolved